plugins/hydra_ax/hydra_plugins/ax_sweeper/ax_sweeper.py

```diff
@@ -0,0 +1,275 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+from typing import Iterator, List, NewType, Tuple
+
+from ax import ParameterType
+from ax.service.ax_client import AxClient
+from hydra._internal.config_search_path import ConfigSearchPath
+from hydra._internal.plugins import Plugins
+from hydra.plugins import SearchPathPlugin, Sweeper
+from omegaconf import OmegaConf
+
+log = logging.getLogger(__name__)
+
+Trial = NewType("Trial", List[str])
+
+# TODO: output directory is overwriting, job.num should be adjusted (depends on issue #284)
+# TODO: Support running multiple random seeds, aggregate mean and SEM
+
+
+class AxSweeperSearchPathPlugin(SearchPathPlugin):
+    """
+    This plugin is allowing configuration files provided by the AxSweeper plugin to be discovered
+    and used once the AxSweeper plugin is installed
+    """
+
+    def manipulate_search_path(self, search_path):
+        assert isinstance(search_path, ConfigSearchPath)
+        # Appends the search path for this plugin to the end of the search path
+        search_path.append("hydra-ax-sweeper", "pkg://hydra_plugins.ax_sweeper.conf")
+
+
+def map_params_to_arg_list(params: dict) -> List[str]:
+    """Method to map a dictionary of params to a list of string arguments"""
+    arg_list = []
+    for key in params:
+        arg_list.append(str(key) + "=" + str(params[key]))
+    return arg_list
+
+
+def yield_batch_of_trials_from_parallelism(
```

Comment:
One scenario where it could be useful is if a call to `ax.get_next_trial()` is expensive. I skimmed through the source of Ax and this does not seem to be the case. So I will push a list-based version.